### PR TITLE
BLADE-333 reorganize gradle tooling model and enable snapshot publishing to liferay nexus

### PR DESCRIPTION
**Pull request by gamerson**

### CODE REVIEW

**Comment by christopherbryanboyd** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: cli/src/main/java/com/liferay/blade/cli/gradle/GradleTooling.java [hunk 1]
```diff
@@ -69,27 +71,35 @@ public static boolean isLiferayModule(File buildDir) throws Exception {
...
+				String initScriptTemplate = FileUtil.collect(GradleTooling.class.getResourceAsStream("init.gradle"));
 
-			libsPath = libsPath.replaceAll("\\\\", "/");
+				String initScriptContents = initScriptTemplate.replaceAll("%files%", files);
 
-			String initScriptContents = initScriptTemplate.replaceAll("%libsPath%", libsPath);
+				Path initPath = tempPath.resolve("init.gradle");
 
-			Path initPath = tempPath.resolve("init.gradle");
+				FileUtil.write(initScriptContents.getBytes(), initPath.toFile());
```
Comment (christopherbryanboyd) [inline cli/src/main/java/com/liferay/blade/cli/gradle/GradleTooling.java]: Should we use `java.nio.file.Files.write(initPath, initScriptContents.getBytes())`?

File: cli/src/test/java/com/liferay/blade/cli/gradle/GradleToolingTest.java [hunk 2]
```diff
@@ -45,14 +45,14 @@
 	public static void setUpClass() throws Exception {
 		File wsDir = temporaryFolder.newFolder("build", "testws1");
 
-		Files.copy(new File("deps.zip").toPath(), _DEPS_ZIP.toPath(), StandardCopyOption.REPLACE_EXISTING);
+		Files.copy(new File("tooling.zip").toPath(), _TOOLING_ZIP.toPath(), StandardCopyOption.REPLACE_EXISTING);
```
[the remark this answers is not in the document]
Comment (gamerson) [reply cli/src/test/java/com/liferay/blade/cli/gradle/GradleToolingTest.java]: Good point.

File: cli/src/main/java/com/liferay/blade/cli/command/ConvertCommand.java [hunk 3]
```diff
@@ -553,19 +553,19 @@ private void _convertToWarProject(File warsDir, File pluginDir) {
 				ivyFile.delete();
 			}
 
-			StringBuilder depsContent = new StringBuilder();
+			StringBuilder toolingContent = new StringBuilder();
 
-			depsContent.append("dependencies {\n");
+			toolingContent.append("dependencies {\n");
```
Comment (christopherbryanboyd) [inline cli/src/main/java/com/liferay/blade/cli/command/ConvertCommand.java]: Perhaps `toolingContent.append("dependencies {" + System.lineSeparator());` for better platform compatibility?

File: cli/src/main/java/com/liferay/blade/cli/command/ConvertCommand.java [hunk 4]
```diff
@@ -553,19 +553,19 @@ private void _convertToWarProject(File warsDir, File pluginDir) {
...
 
-			for (String dep : dependencies) {
-				depsContent.append("\t" + dep + "\n");
+			for (String dependency : dependencies) {
+				toolingContent.append("\t" + dependency + "\n");
```
Comment (christopherbryanboyd) [inline cli/src/main/java/com/liferay/blade/cli/command/ConvertCommand.java]: `System.lineSeparator()` here?

**Pull request merged**